src/System.Threading.Tasks.Extensions/pkg/System.Threading.Tasks.Extensions.pkgproj

```diff
@@ -12,6 +12,7 @@
     <ProjectReference Include="..\src\System.Threading.Tasks.Extensions.csproj" />
 
     <InboxOnTargetFramework Include="netcoreapp2.1" />
+    <InboxOnTargetFramework Include="$(UAPvNextTFM)" />
```

Comment:
@stephentoub I had to revert this part of your recent commit https://github.com/dotnet/corefx/commit/c37e3e5f6b2a3fce69cffe1096ff5c491ed3639d as it broke the uap build.

Reply:
@weshaggard or @ericstj ?

Reply:
This isn't a revert.  @nattress [updated the package index](https://github.com/dotnet/corefx/pull/27805/files#diff-122916076db7087dbc454352fada61eeR4904) to make this inbox (correctly) on UAP**vNext** (AKA uap10.0.16300) just like its inbox on NETCoreApp**vNext** (netcoreapp2.1).
Note that @joperezr will also be updating the package index to correctly account for the old assembly version shipping in the previous version of UAP.  https://github.com/dotnet/corefx/pull/27787

Reply:
those changes are in on master now